openhtf/exe/phase_executor.py

```diff
@@ -122,7 +134,7 @@ def _ThreadProc(self):
     self._phase_outcome = PhaseOutcome(phase_return)
 
   def _ThreadException(self, exc):
```

Comment:
really, `_ThreadException()` should get passed in the `exc_info`, not just the exception instance.  you might modify `threads.py` to do this.  Looks like `threads.py` and this file are the only ones that use it anyway.  then you don't have to do the `sys.exc_info()` here.